packages/lit-dev-content/site/_includes/articles.html

```diff
@@ -0,0 +1,105 @@
+{% extends 'default.html' %}
+
+<!-- When we're rendering API docs, our "versionLinks" data doesn't come from
+     markdown frontmatter, instead it comes from the API JSON data.
+     TODO(aomarks) There must be some way to assign "versionLinks" to
+     "data.otherVersions" from api.html, but I haven't figure out how! -->
+{% if not versionLinks %}
+  {% set versionLinks = data.versionLinks %}
+{% endif %}
+
+{% block head %}
+  {% inlinecss "docs.css" %}
```

Comment:
What about an RSS feed for articles?

Could we just include articles in the existing RSS feed for the blog?

Reply:
Gray says we should only the blog should have RSS

Reply:
We've decided to table this until we get a concensus from the team